templates/eu-cookie-compliance-popup-info--override.html.twig

```diff
@@ -79,10 +79,9 @@
   </div>
 </div>
 
-
 {# This hidden markup is needed by Drupal.eu_cookie_compliance.setPreferenceCheckboxes() #}
 {% if cookie_categories %}
-  <div id="eu-cookie-compliance-categories" class="visually-hidden">
+  <div id="eu-cookie-compliance-categories" style="display: none;">
```

Comment:
Could you please use the "hidden" class instead of inline styling.

Reply:
changed to class='d-none'